tests/Domain/Integrations/Repositories/EloquentIntegrationRepositoryTest.php

```diff
@@ -776,15 +779,23 @@ public static function dataProviderGetIntegrationsThatHaveNotBeenActivatedYet():
             'Should not be selected: Mail already sent' => [
                 IntegrationType::SearchApi,
                 IntegrationStatus::Draft,
-                Carbon::now()->subYears(2),
+                Carbon::now()->subMonths(14),
                 Carbon::now(),
                 true,
                 0,
             ],
+            'Should not be selected: Too old' => [
+                IntegrationType::SearchApi,
+                IntegrationStatus::Draft,
+                Carbon::now()->subMonths(50),
+                Carbon::now(),
```

Comment:
I think this test data is not completely correct. The mail was sent for this one, so you are not sure that the 50 months are taken into account.

Reply:
Updated the test case.